heltour/tournament/templates/tournament/emails/registration_confirmation_subject.txt

```diff
@@ -0,0 +1 @@
+Registration Confirmation - {{ reg.season.league.name }}
```

Comment:
I would prefer calling this "Registration Received" and updating the template names accordingly ("confirmation" is a bit ambiguous).